go/vt/mysqlctl/xtrabackupengine.go

```diff
@@ -49,9 +49,11 @@ type XtrabackupEngine struct {
 var (
 	// path where backup engine program is located
 	xtrabackupEnginePath = flag.String("xtrabackup_root_path", "", "directory location of the xtrabackup executable, e.g., /usr/bin")
-	// flags to pass through to backup engine
+	// flags to pass through to backup phase
 	xtrabackupBackupFlags = flag.String("xtrabackup_backup_flags", "", "flags to pass to backup command. These should be space separated and will be added to the end of the command")
-	// flags to pass through to restore phase
+	// flags to pass through to prepare phase of restore
+	xtrabackupPrepareFlags = flag.String("xtrabackup_prepare_flags", "", "flags to pass to prepare command. These should be space separated and will be added to the end of the command")
```

Comment:
Why space vs comma separated? Is that standard?

Reply:
We pass whatever we get directly to `xtrabackup` and that is how it expects command line params.

Reply:
sure, but you're using strings.Fields to actually pass it in anyway. if we use comma separated, then you can just do the same `strings.Split(a, ",")...` which achieves the same thing. spaces in general seems much more error-prone for flags.

Reply:
this is an excellent point. Since we already have xtrabackup flags that are documented as being space-separated, we would need to implement it in a backwards-compatible way, i.e. allow either spaces or commas and handle both cases.
@morgo @sougou can you chime in if you see any problems with this approach? I checked the xtrabackup reference and it doesn't look like an option value can have a space or comma. Options are specified like `--option_name=option_value`.

Reply:
I think it's unlikely a command will have a space, but they may have a comma. For example, there are several commands which allow a regular expression (which may contain a comma).

Reply:
leave as-is then. i'm totally not blocking. just making sure we have stuff that isn't particularly error-prone, whichever way that may be. sounds like spaces are the right way then.

Reply:
:+1: